specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json

```diff
@@ -0,0 +1,1703 @@
+            "name": "prId",
+            "in": "path",
+            "description": "The stage site identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "202": {
+            "description": "Asynchronous operation in progress."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a static site build": {
+            "$ref": "./examples/DeleteStaticSiteBuild.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/builds/{prId}/functions": {
+      "get": {
+        "tags": [
+          "StaticSites"
+        ],
+        "summary": "Gets the functions of a particular static site build.",
+        "description": "Description for Gets the functions of a particular static site build.",
+        "operationId": "StaticSites_ListStaticSiteBuildFunctions",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the static site.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "prId",
+            "in": "path",
+            "description": "The stage site identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/StaticSiteFunctionOverviewCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the functions of a particular static site build": {
+            "$ref": "./examples/ListStaticSiteBuildFunctions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/config/functionappsettings": {
```

Comment:
i am assuming functionappsettings is an instance of config. looks like config name is static here, do you plan to add more configs that can be PUT? if yes, may be we make this as enum and allowed values for now can be functionAppSettings.

Reply:
this was done to keep things consistent with how Microsoft.Web does app settings for functions today, if possible I'd prefer not to change it